confignode/src/main/java/org/apache/iotdb/confignode/manager/NodeManager.java

```diff
@@ -569,6 +571,23 @@ public Map<Integer, Long> getAllLoadScores() {
     return result;
   }
 
+  public boolean isNodeRemoving(int dataNodeId) {
+    DataNodeHeartbeatCache cache =
+        (DataNodeHeartbeatCache) configManager.getNodeManager().getNodeCacheMap().get(dataNodeId);
+    if (cache != null) {
+      return cache.isRemoving();
+    }
+    return false;
+  }
+
+  public void setNodeRemoving(int dataNodeId) {
```

Comment:
TODO: add a interface,  set node removing is false.   
if  procedure failed and rollback.